src/packages/@ncigdc/containers/ConsequencesTable.js

```diff
@@ -131,6 +136,7 @@ const ConsequencesTableComponent = compose(
       style={{ width: '100%', minWidth: 450 }}
       data={dataRows}
       prefix={paginationPrefix}
+      entityName={'entries'}
```

Comment:
I think the purpose of the the table is to convey "consequences", which is a combination of all the data of its columns. Transcripts also serve to convey consequences. Changing the way rows are aggregated such that transcripts are the only values unique per row doesn't change the purpose of the table

That said, I _am_ also open to renaming this to `Showing 1 - 10 of 27 items`